providers/tests/standard/operators/test_python.py

```diff
@@ -1554,6 +1556,9 @@ def f():
 class TestExternalPythonOperator(BaseTestPythonVirtualenvOperator):
     opcls = ExternalPythonOperator
 
+    def setup_method(self):
+        logging.config.dictConfig(DEFAULT_LOGGING_CONFIG)
```

Comment:
Why is this required now?

Reply:
There are tests which asserts log messages in `caplog.messages`, some how it worked earlier when these operators are in core area, after moving to providers the caplog.messages asserts are failing, so setting up that worked. 

build logs gone unable to retrieve to share.